### ISSUE TITLE
Base 64 image getting created on pasting the image on the clipboard

### ISSUE THREAD

**Issue by skarthikeyan96** [froala/react-froala-wysiwyg#256]
Actual Behaviour 

When pasting the inserted image (S3 url)  in the clipboard, editor creates an image tag with src of `base64` image and then file upload functionality kicks in and uploads the file to the server. Base 64 image which is getting added by the editor in case of large images freezes the browser. 

Expected Behaviour :

Instead of creating the base 64 image , if it can retain the S3 url which it took to create the base 64 image src url, it would be helpful

Version of Froala: 

```json
"react-froala-wysiwyg": "2.9.3",
```

Steps to reproduce:

1. Insert the image in the froala using `imageInsert` command from the editor
2. Copy the inserted image and paste in the clipboard with upload functionality set up. It will upload the image but also appends the base64 encoded image.

**Comment by amitUpscale** [comment]
Yes I also need this desperately. I would like to retain the URL of pasted image instead of encoding it.
